src/app/ui/entries_list/mod.rs

```diff
@@ -160,7 +160,7 @@ impl<'a> EntriesList {
         let items_count = items.len();
 
         let list = List::new(items)
-            .block(self.get_list_block(app.filter.is_some()))
+            .block(self.get_list_block(app.filter.is_some(), app.get_active_entries().count()))
```

Comment:
I think you can use `items_cont` form line 160, without the need to query the entries one more time to get the item_count from it.